CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [0.0.18]
```

Comment:
I didn't update this for the previous versions.

The next version will probably be 0.4.0.

These are the previous versions:


## [0.3.1]

### Bug Fixes
* Add correct package version to url

## [0.3.0]

### Features

* **lock:** Add callback function to `onSessionLocked`

## [0.2.0]

### Features

* Handling of session validation when embedded